po/zh_CN.po

```diff
@@ -282,29 +305,38 @@ msgid ""
 "full length RHS vector such as vector('list',nrow(DT)); i.e., 'plonk' in the "
 "new column."
 msgstr ""
+"赋值语句的右手侧（RHS）指向列 '%s'，其长度为零但并非空值（NULL）。如果"
+"想要删除列请使用空值（NULL）。否则，右手侧必须"
+"长度 > 0，如 NA_integer_。如果想将该列类型修改为"
+"空列表列（list column），正如所有修改列类型的操作一样，请在右手侧提供一个"
+"与该列原数据等长的向量，如 vector('list',nrow(DT))，即，用新数据替换（plonk）重新生成"
+"该列。"
 
```

Comment:
对已有列 '%s' 的赋值其右手侧（RHS）长度为零但并非空值（NULL）
...
如 vector('list',nrow(DT)); 也即用替代品生成新列。

plonk means cheap wine? I think it means generic and fake here but we cannot use too negative words.

Reply:
Different definition of `plonk` 😅 

> set down heavily or carelessly.
> "she plonked her glass on the table"
> sit down heavily and without ceremony.
> "he plonked himself down on the sofa"

Reply:
I get a little bit confused here. Despite the final translation of word `plonk`, `plonk in the new column` refers to the action of recreating the same column, let’s say `col`, using `set(DT, NULL, 'col', value)`, or creating a new column with different name than `col` using `set(DT, NULL, 'newcol', value)`. I think it refers to the first one. If so, the `new column` is **new** because the contents in that column is new. That’s why I use `用新数据替换重新生成该列`. Because from the user’s view, no new column is created but the existing column is replaced by a list of empty lists.

Any insights? @dracodoc

Reply:
I'm not sure about `plonk` meaning now. Previously I thought that means using an empty but similar structured copy to replace, now I'm not sure. For the intention I agree with you it's using new data to replace existing ones.

Reply:
@mattdowle maybe we should add an FAQ about the meaning of `plonk` 😅 

maybe these videos can be a good definition:

https://www.youtube.com/watch?v=zoaGwyYH3as

https://www.youtube.com/watch?v=anJugOiu1jU